sdk/personalizer/Azure.AI.Personalizer/src/Generated/Models/PersonalizerSlotResult.cs

```diff
@@ -34,6 +34,6 @@ internal PersonalizerSlotResult(string slotId, string rewardActionId)
             RewardActionId = rewardActionId;
         }
         /// <summary> RewardActionID is the action ID recommended by Personalizer. </summary>
-        public string RewardActionId { get; }
+        public string RewardActionId { get; set; }
```

Comment:
has to be updated through PersonalizerSlotResult in Models instead of directly adding it in Generated folder class

Reply:
I removed the setter.